pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -219,6 +218,10 @@ type GracefulEvictionTask struct {
 	// +required
 	Producer string `json:"producer"`
 
+	// Evicted indicates if the eviction is performed.
+	// +required
+	Evicted bool `json:"evicated"`
```

Comment:
Does this mean that the evicted elements will also continue to exist in the queue, and when will they be removed from the current queue?

Reply:
> Does this mean that the evicted elements will also continue to exist in the queue

Yes

> when will they be removed from the current queue?

graceful eviction controller detects (RB/CRB).spec.evictionTasks[x].evicated with true, it will delete it from (RB/CRB).spec.evictionTasks.